src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -565,6 +565,21 @@ private static void PartialMethodChecks(SourceOrdinaryMethodSymbol definition, S
                 hasTypeDifferences = true;
             }
 
+            if (SourceMemberContainerTypeSymbol.CheckRefReadonlyInMismatch(
+                constructedDefinition, implementation, diagnostics,
+                static (diagnostics, _, implementation, implementationParameter, _, arg) =>
+                {
+                    var (definitionParameter, _) = arg;
+                    var location = implementation.GetFirstLocation();
+                    // Modifier of parameter '{0}' doesn't match the corresponding parameter '{1}' in partial declaration.
+                    diagnostics.Add(ErrorCode.WRN_PartialDifferentRefness, location, definitionParameter, implementationParameter);
+                },
+                extraArgument: (object)null,
+                invokedAsExtensionMethod: false))
+            {
+                hasTypeDifferences = true;
```

Comment:
>hasTypeDifferences = true; [](http://example.com/codeflow?start=16&length=26)

Are we testing effect of this assignment?

Reply:
Adding test `PartialMembers_In_DifferentNullability` that would fail if this assignment wasn't present.